Topic 5/src/bootcamp/exercises/MySql.java

```diff
@@ -0,0 +1,40 @@
+package bootcamp.exercises;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+
+public class MySql{	
+
+	public void ListPerCourses (String nameCourse) throws Exception{
+		
+		Class.forName("com.mysql.jdbc.Driver");
+		
+		Connection con = DriverManager
+		          .getConnection("jdbc:mysql://localhost:3306/highschool", "root", "14929952");
+		PreparedStatement statement= con.prepareStatement("select t.first_name, t.last_name, s.first_name, s.last_name from teacher t, course c, student s, assists a where c.assigned_teacher= t.id_teacher and c.id_course= a.id_course and a.id_student= s.id and c.name='" + nameCourse+"' order by s.last_name;");
+		ResultSet result= statement.executeQuery();
+		if(result.next()){
```

Comment:
[High] Remember that *next* method means: Moves the ResultSet to point at the next row in the ResultSet. 
So, in this case you're not printing the first row information (just teacher's information).